src/test/java/net/kaczmarzyk/spring/data/jpa/utils/SpecificationBuilderTest.java

```diff
@@ -58,35 +58,35 @@ public class SpecificationBuilderTest extends IntegrationTestBase {
 	public interface CustomSpecification extends Specification<Customer> {
 	}
 
-	@Join(path = "orders", alias = "o")
-	@Join(path = "o.tags", alias = "t", type = JoinType.INNER)
+	@Join(path = "orders", alias = "o", type = JoinType.INNER)
+	@Join(path = "o.tags", alias = "t", type = JoinType.LEFT)
 	@Or({
 			@Spec(path = "o.itemName", pathVars = "orderIn", spec = In.class),
 			@Spec(path = "t.name", headers = "tag", spec = Equal.class)
 	})
 	public interface CustomSpecificationWithPathVar extends Specification<Customer> {
 	}
 
-	@Join(path = "orders", alias = "o")
-	@Join(path = "o.tags", alias = "t", type = JoinType.INNER)
+	@Join(path = "orders", alias = "o", type = JoinType.INNER)
+	@Join(path = "o.tags", alias = "t", type = JoinType.LEFT)
```

Comment:
changed to left, as there are no tags and with eager evaluation of inner join the result was empty